sematic/ui/packages/main/src/pipelines/BasicMetricsPanel.tsx

```diff
@@ -0,0 +1,88 @@
+import { Run } from "@sematic/common/src/Models";
+import { usePipelineRunContext } from "src/hooks/pipelineHooks";
+import useBasicMetrics from "src/hooks/metricsHooks";
+import Loading from "src/components/Loading";
+import { Alert, Box, Grid, Table, TableBody, TableCell, TableRow, Tooltip, Typography, useTheme } from "@mui/material";
+import { useMemo } from "react";
+import {durationSecondsToString} from "src/utils";
+import CalculatorPath from "src/components/CalculatorPath";
+import HelpIcon from "@mui/icons-material/Help";
+import { BasicMetricsPayload } from "src/Payloads";
+
+function TopMetric(props: {value: string, label: string, help?: string}) {
+  const {value, label, help } = props;
+  const theme = useTheme();
+
+return <Box sx={{textAlign: "center", p: 10}}>
+    <Typography sx={{fontSize: 70, fontWeight: 500, lineHeight: 1}}>{value}</Typography>
+    <Box sx={{display: "flex", flexDirection: "row", alignItems: "center", columnGap: 1, justifyContent: "center"}}>
+    <Typography sx={{fontSize:20}}>
+      {label}
+    </Typography>
+    {help !== undefined && <Tooltip title={help}>
+      <HelpIcon sx={{color: theme.palette.grey[300]}} fontSize="small"/>
+    </Tooltip>}
+    </Box>
+</Box>
+
+}
+
+
+export function runSuccessRate(payload: BasicMetricsPayload, run: Run) : string {
+  const totalCount = payload.content.total_count;
+  const percentRate = (payload.content.count_by_state["RESOLVED"] || 0) / (totalCount || 1) * 100;
+  return `${Math.round(percentRate)}%`;
```

Comment:
Please use floor in order to not show a clean 100% when fewer than 1% of runs are failures?